USER/userIntro.sqf

```diff
@@ -0,0 +1,27 @@
+GRAD_introCam_playOnStart = true;							//Ob das das intro bei Spielstart automatisch abgespielt werden soll
+GRAD_introCam_allowForJIP = false;							//Ob JIP Spieler das Intro sehen sollen
+
+/*
+	SHOT TYPES:
+	["CAMERA", duration, pos, target, fov (optional), showBorders (optional), fade (optional), fadeTime (optional), fadeText (optional), effectCam (optional), effectCamType (optional)],
+	["MOVE", duration, pos, target, fov (optional), offset (optional), commitTime (optional)],
+	["ATTACH", duration, object, target, fov (optional), offset (optional)],
+	["ROTATE", duration, target, height (optional), radius (optional), startingAngle (optional), endAngle (optional), clockwise (optional), rise (optional)]
+    ["BLEND",  duration, textType (optional), blendOut (optional), blendIn (optional), extension]
+
+* CAM PARAMETERS:
+* Duration <NUMBER>
+* Positions <OBJECT / POSITION>
+* Target <OBJECT / POSITION>
+* Zoom Factor <NUMBER> (default: 1)
+* Show Borders <BOOLEAN> (default: true)
+* Fade <NOOLEAN> (default: true)
+* Fade Time <NUMBER> (default: 1)
+* Fade Text <STRING> (default: "")
+* Camera Effect <STRING> (default: "internal")
+* Camera Effect Type <STRING> (default: "BACK")
+*/
+
+GRAD_introCam_shotDefinitions = [
+
+];
```

Comment:
USER\userIntro.sqf muss noch in initPlayerLocal.sqf eingebunden werden?

Reply:
Ne, brauch es nicht. Wird direkt in die benötigten Files geladen

Reply:
Ach ist jetzt hier hardgecoded? Soll grad-introCam dann so npm-released werden?
https://github.com/gruppe-adler/CO_Template.VR/pull/62/files#diff-e02fb505878453b8e14e481fa34b83fb9a902f05c9344452c22d5072785889bcR5

Reply:
Hatte nie die intention das auf npm zu packen. 
Ist das denn notwendig?

Reply:
wenn du's in node_modules reinpackst, dann muß es auch auf npm sein.

Reply:
ohne npm ist die lage in node_modules sinnfrei? 

wenn es noch woanders benutzt werden soll als im template bietet sich npm als update mangement schon an. klappt bei den anderen modulen ja auch super.

Reply:
Dann schau ich mir nachher mal an wie das geht